<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Z3D::ZCameraSettingsWidget</class>
 <widget class="QWidget" name="Z3D::ZCameraSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4">
    <widget class="QPushButton" name="savePresetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save preset</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="loadPresetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Load preset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QVBoxLayout" name="propertyBrowserLayout"/>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
